Delete_User.css

```diff
@@ -76,11 +93,18 @@ ul li:hover ul{
 	visibility: visible;
 }
 
+ul li:hover #a{
```

Comment:
This code use to display the drop down menu of the setting column when I hover on Setting.